scan/tasks.py

```diff
@@ -49,6 +53,35 @@ def task_cmd():
         if snr_master :
             logger.info("SNR Master Data Received")
 
+    """
+    ######### Update Bootstrap Peers ###########
+        Best to run this task instead of having the page load it each time it's opened. 
+        Also exports to env for peer scan use. 
+    """
+    if AUTO_BOOTSTRAP_PEERS:
+        bootstrap_peers = BRS_BOOTSTRAP_PEERS
+        caching_peers = CachingBootstrapNodes()
+        bootstrap_network = os.environ.get("BRS_BOOTSTRAP_NETWORK", ".signum.network")
+        if bootstrap_network is "": bootstrap_network = ".signum.network"
```

Comment:
![image](https://github.com/signum-network/signum-explorer/assets/57628432/e9a85fbd-3a74-43c5-8dc3-32b2febe5ea5)

Reply:
The second argument in os.environ.get is the default value. So `if bootstrap_network is "": bootstrap_network = ".signum.network"` doesn't make sense.
https://docs.python.org/3/library/os.html#os.getenv
Example: If BRS_BOOTSTRAP_NETWORK no in .env or equal "", then BRS_BOOTSTRAP_NETWORK = ".signum.network"

Reply:
> The second argument in os.environ.get is the default value. So `if bootstrap_network is "": bootstrap_network = ".signum.network"` doesn't make sense. https://docs.python.org/3/library/os.html#os.getenv Example: If BRS_BOOTSTRAP_NETWORK no in .env or equal "", then BRS_BOOTSTRAP_NETWORK = ".signum.network"

You're correct now that I see it, not sure why mine doesn't throw that error though. Will fix..